OpenDreamRuntime/Objects/Types/DreamObjectWorld.cs

```diff
@@ -58,7 +58,7 @@ private bool DisplayIPv6 {
     }
 
     /// <summary> Tries to return the address of the server, as it appears over the internet. May return null.</summary>
-    private IPAddress? InternetAddress => null; //TODO: Implement this!
+    private IPAddress? InternetAddress => _netManager.ServerChannel?.RemoteEndPoint.Address ?? null;
```

Comment:
I think this only tells you the local address this server is bound to, not the external IP others will have to use to connect.

I don't know if RT has a way to get this currently.